### DISCUSSION TITLE
Where are the SCORM packages stored ?

### DISCUSSION THREAD

**Discussion by Moigaston** [chamilo/chamilo-lms#5851] [General]
The question is in the title. I mean, SCORM packages are not copied either when copying a course containing SCORM packages or during a backup. Where should I search if I want to have them included in a copied course? I didn't find any SCORM packages anywhere in the Chamilo database.

**Comment by NicoDucou** [comment]
Hi @Moigaston 
By default scorm package included in course export are not imported in Chamilo.
If you want the scorm package to be imported you need to add this setting in app/config/configuration.php :
```
// Allow SCORM packages when importing a course
// $_configuration['allow_import_scorm_package_in_course_builder'] = false;
```
Best regards

**Reply by Moigaston** [reply]
Thanks a lot, NicoDucou!

**Comment by Moigaston** [comment]
Unfortunately, it doesn't work. Uncommenting the lines causes Chamilo to stop working.
[url=https://postimages.org/][img]https://i.postimg.cc/yNKBtSnF/config-php.png[/img][/url]
[url=https://postimages.org/][img]https://i.postimg.cc/BQ10vCVT/page-ne-fonctionne-pas.png[/img][/url]
